pkg/services/sqlstore/user_test.go

```diff
@@ -175,6 +175,30 @@ func TestUserDataAccess(t *testing.T) {
 					So(found, ShouldBeTrue)
 				})
 			})
+
+			Convey("When batch disabling users", func() {
+				userIdsToDisable := []int64{}
+				for i := 0; i < 3; i++ {
+					userIdsToDisable = append(userIdsToDisable, users[i].Id)
+				}
+				disableCmd := m.BatchDisableUsersCommand{UserIds: userIdsToDisable, IsDisabled: true}
+
+				err = BatchDisableUsers(&disableCmd)
+				So(err, ShouldBeNil)
+
+				Convey("Should disable all provided users", func() {
+					query := m.SearchUsersQuery{}
+					err = SearchUsers(&query)
+
+					So(query.Result.TotalCount, ShouldEqual, 5)
+					So(query.Result.Users[0].IsDisabled, ShouldBeTrue)
```

Comment:
This test depends on the returned order of users from the database unless I'm mistaken. From what I could tell we don't have any ordering setup when doing the query, so it seems like a bad idea to depend on the order (and is probably the reason why the Postgres integration tests are failing).